[Workers] Standard pricing launch (#11575)
cloudflare/cloudflare-docs: content/workers/wrangler/configuration.md

  - The usage model of your Worker. Refer to [usage models](/workers/platform/pricing/#workers).

  {{<Aside type="note">}}

>   The usage model set in a script's wrangler.toml will be ignored after an account has opted-in to [Workers Standard](/workers/platform/pricing/#workers) pricing. It must be configured through the dashboard (Workers & Pages > Select your Worker > Settings > Usage Model). 

Reviewer: ```suggestion
 After you have opted into the the [Workers Standard](/workers/platform/pricing/#workers) usage model, the usage model configured in your Worker's `wrangler.toml` will be ignored . It must instead be configured through the dashboard (Workers & Pages > Select your Worker > Settings > Usage Model).
```
